commands/google-maps/google-maps.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Search Google Maps
+# @raycast.mode silent
+# @raycast.packageName Google Maps
+
+# Optional parameters:
+# @raycast.icon google-map.png
+# @raycast.argument1 { "type": "text", "placeholder": "Location", "optional": true}
+# @raycast.argument2 { "type": "text", "placeholder": "Destination", "optional": true}
+
+first_argument=${1// /+}
+second_argument=${2// /+}
```

Comment:
I think the naming of these variables could be a bit more clear/intuitive:
```suggestion
origin=${1// /+}
destination=${2// /+}
```

Reply:
Much, much better than my suggestion! 😅